refinery/analysis_manager/utils.py

```diff
@@ -194,19 +163,17 @@ def create_noderelationship_analysis(validated_analysis_config):
 
     # ANALYSIS MODEL
     # How to create a simple analysis object
-    if not custom_name:
-        temp_name = "{} {}".format(
+    if not name:
+        name = "{} {}".format(
```

Comment:
Is there logic that should be shared between here and line 81? (This feels like copy-and-paste which preceded this commit, so not a high priority to fix.)